cgmanifest.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I am not sure if I can add this file into public Github repo.

Reply:
Removed in revision